src/gotoDefinition.ts

```diff
@@ -48,4 +49,14 @@ export const registerGotoDefinition = () => {
             return [{ range, targetRange: range, uri, targetUri: uri }]
         },
     })
+    const setReferenceButtonVisibility = () => {
+        const isMenuButtonEnabled = getExtensionSetting('enableFindReferencesButton')
+        void vscode.commands.executeCommand('setContext', getExtensionCommandId('findComponentReferences'), isMenuButtonEnabled)
```

Comment:
See vscode docs: https://code.visualstudio.com/api/extension-guides/command#using-a-custom-when-clause-context. Commands and [when clauses](https://code.visualstudio.com/api/references/when-clause-contexts) have nothing in common.

For now, we don't have typed wrappers for when contexts, so you can use previous variant.

```suggestion
        void vscode.commands.executeCommand('setContext', `veturExtended.enableFindReferencesButton`, isMenuButtonEnabled)
```

Alternatively you can use `process.env.IDS_PREFIX!` instead of `veturExtended`, however it doesn't really matter here.

Reply:
Decided to leave `veturExtended` prefix